decode.rb

```diff
@@ -0,0 +1,38 @@
+@decoder_letters = { '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G',
+                     '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N',
+                     '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U',
+                     '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z', '.----' => '1',
+                     '..---' => '2', '...--' => '3', '....-' => '4', '.....' => '5', '-....' => '6', '--...' => '7',
+                     '---..' => '8', '----.' => '9', '-----' => '0' }
+
+# method to decode a char
+def decode_char(string)
+  @decoder_letters.fetch(string)
+end
+
+puts decode_char('.-')
+
+# Method to decode the word
+def decode_word(string)
+  output = ''
+  string
+    .split
+    .each do |i|
+    output += decode_char(i)
+  end
+  output
+end
+
+puts decode_word('-- -.--')
```

Comment:
- [ ] Hi @myelin0, if you check once more the project requirements, this method returns **"MY"** and NOT **MY**. Kindly make sure that the word is surrounded with quotes.